p8e-api/src/main/kotlin/io/provenance/engine/stream/ScopeStream.kt

```diff
@@ -49,36 +49,9 @@ class ScopeStream(
     // The current event stream ID
     private val eventStreamId = java.util.UUID.fromString(eventStreamProperties.id)
 
-    // The event stream service uri
-    private val uri = URI(eventStreamProperties.uri)
```

Comment:
This reminded me we need to remove this from the properties files if it wasn't already 😉

Reply:
oh yeah, additionally, I need to add in the other new urls to the non-local files